R/utils.R

```diff
@@ -367,3 +367,40 @@ set_num_threads <- function(num_threads) {
          call. = FALSE)
   }
 }
+
+check_divergences <- function(data_csv) {
+  divergences <- posterior::extract_one_variable_matrix(data_csv$sampler, "divergent__")
+  num_of_divergences <- sum(divergences)
+  if (num_of_divergences > 0) {
+    if(data_csv$sampling_info$sample_save_warmup == 1) {
```

Comment:
Don't report divergences during warmup. These happen a lot cause the timestep is adjusting (divergences will cause the timestep to go down).